dockerdata/dnsdist.conf

```diff
@@ -0,0 +1,9 @@
+setLocal('0.0.0.0')
+
+resolver = require 'dnsdist-resolver'
+resolver.servers.auth = {pool='auths'}
+resolver.servers.recursor = {pool='recursors'}
```

Comment:
This hardcodes the docker-compose setup. We probably need a more generic way of doing this, like passing the pools in env vars. At the very least add comments, because it's not obvious what the hardcoded assumptions are here.

For that static setup using [depends_on](https://docs.docker.com/compose/compose-file/#depends_on) in `docker-compose.yml` and `os.getenv` here would probably suffice, but this approach could be useful in other setups.